Makefile

```diff
@@ -11,6 +11,7 @@
 SA?=source activate
 ENV:=enterprise-gateway-dev
 SHELL:=/bin/bash
+MULTIARCH_BUILD?=
```

Comment:
Does undefined resolves to false, and thus you don't need to initialize this to false?

Reply:
yes